svf-llvm/include/SVF-LLVM/SVFIRBuilder.h

```diff
@@ -97,6 +97,7 @@ class SVFIRBuilder: public llvm::InstVisitor<SVFIRBuilder>
     /// GetObject - Return the object node (stack/global/heap/function) according to a LLVM Value
     inline NodeID getObjectNode(const Value* V)
     {
+        V = LLVMUtil::getGlobalRep(V);
```

Comment:
Global rep should always be a global. The getObject node can return a null global obj. Why adding this line?

Reply:
The reason why I add `V = LLVMUtil::getGlobalRep(V);` is that `val` is converted to global representation in `collectObj` to perform `insert` on `symInfo->objSymMap`
https://github.com/SVF-tools/SVF/blob/0eae65388008dbfafc0287873bfb4c8819582183/svf-llvm/lib/SymbolTableBuilder.cpp#L318-L330

Reply:
Let me give you a case

Reply:
Stack is like this (pictures below may be more vivid), `objSymMap` is updated in above `collectObj`
![image](https://github.com/SVF-tools/SVF/assets/35480440/4d6452db-adaa-4c7a-ac1a-7d91def12362)
![image](https://github.com/SVF-tools/SVF/assets/35480440/39c60ab4-d4eb-4d04-9150-d87c63a22cc2)
![image](https://github.com/SVF-tools/SVF/assets/35480440/4ccad5f0-56e5-436f-8677-c0de499b993f)

Reply:
But I'm not sure whether this fix is accurate or not